pkg/registry/batch/cronjob/storage/storage.go

```diff
@@ -124,10 +124,17 @@ func (r *InstantiateREST) Create(ctx genericapirequest.Context, name string, obj
 		return nil, fmt.Errorf("got object of type %T, expected *batch.CronJob", sourceCronJobObj)
 	}
 
+	// add an extra label onto the created job to signal to the cronjob controller that it should adopt this job
+	jobLabels := make(map[string]string)
+	for _, key := range sourceCronJob.Spec.JobTemplate.Labels {
+		jobLabels[key] = sourceCronJob.Spec.JobTemplate.Labels[key]
+	}
+	jobLabels["createdByInstantiate"] = "yes"
```

Comment:
Annotation will be better for this type of information, name it something like `cronjob.kubernetes.io/instantiate=manual`

Reply:
@kubernetes/api-reviewers thoughts?

Reply:
Comment still applies.